skin.json

```diff
@@ -89,24 +90,6 @@
     "countDownTime": "10"
   },
   "shareScreen": {
-    "share": [
-      {
-        "buttonName": "Facebook",
-        "icon": {
-          "fontFamilyName": "fontawesome",
-          "fontString": "\uf230",
-          "fontStyleClass": "icon icon-facebook"
-        }
-      },
-      {
-        "buttonName": "Twitter",
-        "icon": {
-          "fontFamilyName": "fontawesome",
-          "fontString": "\uf099",
-          "fontStyleClass": "icon icon-twitter"
-        }
-      }
-    ],
```

Comment:
I think PBA had been working with these, do we have their signoff to remove them?